tributors/main/parsers/codemeta.py

```diff
@@ -63,7 +63,7 @@ def update(self, thresh=1, from_resources=None, save=True):
 
     def update_metadata(self):
         """Update codemeta metadata from the repository, if we can."""
-        self.data["keywords"] = self.repo.topics(self.data["keywords"])
+        self.data["keywords"] = self.repo.topics(self.data.get("keywords", None))
```

Comment:
might better be
```suggestion
        self.data["keywords"] = self.repo.topics(self.data.get("keywords", []))
```
since from a quick look - needs a `list`.

Reply:
`self.repo.topics` takes `None` as default - but `[]` is probably more explicit

Reply:
changed